fix build with gcc 10

Define quiet as static to avoid the following build failure with gcc 10
(which defaults to -fno-common):

/home/naourr/work/instance-2/output-1/host/lib/gcc/arm-buildroot-linux-gnueabihf/10.2.0/../../../../arm-buildroot-linux-gnueabihf/bin/ld: CMakeFiles/get-edid.dir/i2c.c.o:(.bss+0x0): multiple definition of `quiet'; CMakeFiles/get-edid.dir/get-edid.c.o:(.bss+0x0): first defined here

Fixes:
 - http://autobuild.buildroot.org/results/28bde8049b6610273bceae26eca407c819a37dcd

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: sent to pyrophobicman@gmail.com]

diff -Nura read-edid-3.0.2.orig/get-edid/classic.c read-edid-3.0.2/get-edid/classic.c
--- read-edid-3.0.2.orig/get-edid/classic.c	2020-09-25 23:25:01.928805414 +0200
+++ read-edid-3.0.2/get-edid/classic.c	2020-09-25 23:26:37.440807301 +0200
@@ -26,7 +26,7 @@
 #define dosmemput(buffer,length,offset) memcpy(offset,buffer,length)
 
 #define display(...) if (quiet == 0) { fprintf(stderr, __VA_ARGS__); }
-int quiet;
+static int quiet;
 
 real_ptr far_ptr_to_real_ptr( uint32 farptr )
 {
diff -Nura read-edid-3.0.2.orig/get-edid/get-edid.c read-edid-3.0.2/get-edid/get-edid.c
--- read-edid-3.0.2.orig/get-edid/get-edid.c	2020-09-25 23:25:01.928805414 +0200
+++ read-edid-3.0.2/get-edid/get-edid.c	2020-09-25 23:28:20.740809341 +0200
@@ -9,7 +9,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-int quiet=0;
+static int quiet=0;
 int i2conly=0; //0=both, 1=i2conly, 2=classiconly
 int i2cbus=-1;
 int classmon=0;
diff -Nura read-edid-3.0.2.orig/get-edid/i2c.c read-edid-3.0.2/get-edid/i2c.c
--- read-edid-3.0.2.orig/get-edid/i2c.c	2020-09-25 23:25:01.928805414 +0200
+++ read-edid-3.0.2/get-edid/i2c.c	2020-09-25 23:28:32.152809567 +0200
@@ -15,7 +15,7 @@
 
 //Ideas (but not too much actual code) taken from i2c-tools. Thanks guys.
 
-int quiet;
+static int quiet;
 
 #define display(...) if (quiet == 0) { fprintf(stderr, __VA_ARGS__); }
 
